Prog/BPKPresentationResults.sas

```diff
@@ -0,0 +1,494 @@
+/**************************************************************************
+ Program:  BPKPresentationResults.sas
+ Library:  BridgePk
+ Project:  NeighborhoodInfo DC
+ Author:   J.Dev
+ Created:  3/20/17
+ Version:  SAS 9.2
+ Environment:  Local Windows session
+ 
+ Description:  Prepare data for initial 11th Street Bridge Park Data Room
+			   Presentation on 3/27/2017, from ACS, Realprop, and Police
+
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( ACS )
+%DCData_lib( Realprop )
+%DCData_lib( Police )
+%DCData_lib( NCDB )
+
+libname Bridgepk "L:\Libraries\Bridgepk\Data\";
```

Comment:
You shouldn't need to use a libname statement here. You should use %DCData_lib() as with the other libraries.